src/modules/cpu/kernel/crop_mirror_normalize.hpp

```diff
@@ -79,12 +76,11 @@ RppStatus crop_mirror_normalize_u8_u8_host_tensor(Rpp8u *srcPtr,
                         dstPtrTempG += vectorIncrementPerChannel;
                         dstPtrTempB += vectorIncrementPerChannel;
                     }
-
                     for (; vectorLoopCount < bufferLength; vectorLoopCount += 3)
                     {
-                        *dstPtrTempR = (Rpp8u) RPPPIXELCHECK((((Rpp32f) (srcPtrTemp[0] - mean[0]) * invStdDev[0])));
-                        *dstPtrTempG = (Rpp8u) RPPPIXELCHECK((((Rpp32f) (srcPtrTemp[1] - mean[1]) * invStdDev[1])));
-                        *dstPtrTempB = (Rpp8u) RPPPIXELCHECK((((Rpp32f) (srcPtrTemp[2] - mean[2]) * invStdDev[2])));
+                        *dstPtrTempR = (Rpp8u) RPPPIXELCHECK((((Rpp32f) (srcPtrTemp[0] * multiplierTensor[incrementPerImage]) + offsetTensor[incrementPerImage])));
```

Comment:
Please don't compute incrementPerImage + X inside the loop n times. Store in an array say cmnParamLocs[3] in advance if needed.